.github/workflows/dotnet-core.yml

```diff
@@ -12,11 +12,11 @@ jobs:
     runs-on: ubuntu-latest
 
     steps:
-    - uses: actions/checkout@v2
+    - uses: actions/checkout@v4
     - name: Setup .NET Core
-      uses: actions/setup-dotnet@v1
+      uses: actions/setup-dotnet@v4
       with:
-        dotnet-version: 6.0.201
+        dotnet-version: 8
```

Comment:
Why did you change the .NET version?

Reply:
@TysonMN 
Because .NET 6 reached Out Of Support last year (in November, if I remember correctly).
.NET 8 is the next LTS which is supported by the end of 2026.

Reply:
So it wasn't needed to add this feature then?

If not, then please make a separate PR with a single commit that upgrades .NET.  In the commit message body, put the explanation that you have given here.

Reply:
I can check, but I believe that it won't compile under .NET 6.
I can make a PR so we can upgrade to .NET 8 first, and then rebase this one on top.

Reply:
I mean, .NET 8 is needed for the feature the way it is now. 
I just decided not to take any extra effort to add .NET 6 workarounds and simply use the officially supported version.

This way we can say that it is needed to add the feature.

Reply:
> I can make a PR so we can upgrade to .NET 8 first, and then rebase this one on top.

Yes. Two PRs in that order would be good.

Reply:
@TysonMN 
OK, let's merge https://github.com/hedgehogqa/fsharp-hedgehog-experimental/pull/79 and then I'll rebase this one on top

Reply:
@TysonMN As discussed, I have rebased this branch on top of .NET 8 enabled `master`.
Please check again.